scripts/ts-node/register.js

```diff
@@ -1,9 +1,6 @@
 const tsNode = require('ts-node');
 
 tsNode.register({
-  transpileOnly: true,
-  // https://github.com/TypeStrong/ts-node#cwdmode - consume tsconfig.json within scripts/
-  cwd: __dirname,
   // https://github.com/TypeStrong/ts-node#skipproject - don't read tsconfig within ts-node
   skipProject: true,
   swc: true,
```

Comment:
Wondering if we can remove this completely and just use config in `tsconfig.json` (https://typestrong.org/ts-node/docs/swc/). Or this will be too implicit?